openreview/venue/matching.py

```diff
@@ -315,14 +315,20 @@ def _build_note_conflicts(self, submissions, user_profiles, get_profile_info, co
         ## for AC conflicts, check SAC conflicts too
         sac_user_info_by_id = {}
         if self.is_area_chair:
-            sacs_by_ac =  { g['id']['head']: [v['tail'] for v in g['values']] for g in self.client.get_grouped_edges(invitation=self.venue.get_assignment_id(self.senior_area_chairs_id), groupby='head', select=None)}
+            sacs_by_ac =  { g['id']['head']: [v['tail'] for v in g['values']] for g in self.client.get_grouped_edges(invitation=self.venue.get_assignment_id(self.senior_area_chairs_id, deployed=True), groupby='head', select=None)}
```

Comment:
I realized we have been getting the `Proposed_Assignment` edges here, not the `Assignment` ones